src/imitation/scripts/parallel.py

```diff
@@ -192,14 +192,10 @@ def inner(config: Mapping[str, Any], reporter) -> Mapping[str, Any]:
         ex.observers = [FileStorageObserver("sacred")]
 
         # Apply base configs to get modified `named_configs` and `config_updates`.
-        named_configs = []
-        named_configs.extend(base_named_configs)
-        named_configs.extend(run_kwargs["named_configs"])
+        named_configs = [*base_named_configs, *run_kwargs["named_configs"]]
```

Comment:
I don't mind using addition instead of unpacking. I have a slight preference for unpacking as it's an operation that has become quite intuitive to me, but I understand that for most people the addition operator is probably the most obvious.